docs/en/advanced/Command-Line-Interface.md

```diff
@@ -29,6 +11,253 @@ priority: 0
 Alluxio's command line interface provides users with basic file system operations. You can invoke
 the following command line utility to get all the subcommands:
 
+```bash
+$ ./bin/alluxio
+Usage: alluxio [COMMAND]
+       [format [-s]]
+       [getConf [key]]
+       [logLevel]
+       [runTests]
+       ...
+```
+
+## General operations
+
+In this section, we list the usages and examples of general Alluxio operations except file system 
+and file system admin operations. File system operations will be listed in the next section 
```

Comment:
The table of contents already shows that fs operations are listed in the next section, we don't need to repeat it here

Reply:
Removed this mention